project2/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+
+
+
+
+
+
+<body>
+
+
+
+
+
+    <div class="card">
+        <img src="Healthy-Garlic-Shrimp-Pasta-with-Lemon.jpg" alt="">
```

Comment:
You should add `alt` text describing your photo:
```suggestion
        <img src="Healthy-Garlic-Shrimp-Pasta-with-Lemon.jpg" alt="A delicious blend of shrimp, vegetables and spaghetti noodles.">
```

Reply:
Remember to resize and compress your images before calling the project done:
<img width="1032" alt="Screen Shot 2019-10-28 at 10 42 35 AM" src="https://user-images.githubusercontent.com/2653980/67688135-bed7a380-f96f-11e9-8f49-86771ec6897b.png">